llvm.spec

```diff
@@ -38,6 +38,7 @@ cd %{_builddir}/build
 
 cmake %{_builddir}/llvm-%{realversion}-%{llvmCommit}/llvm \
   -G Ninja \
+  -DGCC_INSTALL_PREFIX="${GCC_ROOT}" \
```

Comment:
@fwyzard , looks like building llvm with `GCC_INSTALL_PREFIX`  fixed the llvm not finding our gcc header issue. After this change now I see that `clang++` shows the correct gcc paths
```
>  clang++ -E -x c++ - -v
clang version 14.0.6
Target: x86_64-redhat-linux-gnu
Thread model: posix
InstalledDir: /data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc10/external/llvm/14.0.6-7e877bac6aa04ac511ab1c515ec417ab/bin
Found candidate GCC installation: /data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc10/external/gcc/10.3.0-84898dea653199466402e67d73657f10/lib/gcc/x86_64-redhat-linux-gnu/10.3.0
Selected GCC installation: /data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc10/external/gcc/10.3.0-84898dea653199466402e67d73657f10/lib/gcc/x86_64-redhat-linux-gnu/10.3.0
Candidate multilib: .;@m64
Selected multilib: .;@m64
Found CUDA installation: /cvmfs/cms-ib.cern.ch/nweek-02748/el8_amd64_gcc10/external/cuda/11.5.2-c927b7e765e06433950d8a7eab9eddb4, version 11.5
 (in-process)
 "/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc10/external/llvm/14.0.6-7e877bac6aa04ac511ab1c515ec417ab/bin/clang-14" -cc1 -triple x86_64-redhat-linux-gnu -E -disable-free -clear-ast-before-backend -disable-llvm-verifier -discard-value-names -main-file-name - -mrelocation-model static -mframe-pointer=all -fmath-errno -ffp-contract=on -fno-rounding-math -mconstructor-aliases -funwind-tables=2 -target-cpu x86-64 -tune-cpu generic -debugger-tuning=gdb -v -fcoverage-compilation-dir=/data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_12_6_CLANG_X_2022-08-31-2300 -resource-dir /data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc10/external/llvm/14.0.6-7e877bac6aa04ac511ab1c515ec417ab/lib64/clang/14.0.6 -internal-isystem /data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc10/external/gcc/10.3.0-84898dea653199466402e67d73657f10/lib/gcc/x86_64-redhat-linux-gnu/10.3.0/../../../../include/c++/10.3.0 -internal-isystem /data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc10/external/gcc/10.3.0-84898dea653199466402e67d73657f10/lib/gcc/x86_64-redhat-linux-gnu/10.3.0/../../../../include/c++/10.3.0/x86_64-redhat-linux-gnu -internal-isystem /data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc10/external/gcc/10.3.0-84898dea653199466402e67d73657f10/lib/gcc/x86_64-redhat-linux-gnu/10.3.0/../../../../include/c++/10.3.0/backward -internal-isystem /data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc10/external/llvm/14.0.6-7e877bac6aa04ac511ab1c515ec417ab/lib64/clang/14.0.6/include -internal-isystem /usr/local/include -internal-isystem /data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc10/external/gcc/10.3.0-84898dea653199466402e67d73657f10/lib/gcc/x86_64-redhat-linux-gnu/10.3.0/../../../../x86_64-redhat-linux-gnu/include -internal-externc-isystem /include -internal-externc-isystem /usr/include -fdeprecated-macro -fdebug-compilation-dir=/data/cmsbld/jenkins/workspace/ib-run-pr-tests/CMSSW_12_6_CLANG_X_2022-08-31-2300 -ferror-limit 19 -fgnuc-version=4.2.1 -fcxx-exceptions -fexceptions -fcolor-diagnostics -faddrsig -D__GCC_HAVE_DWARF2_CFI_ASM=1 -o - -x c++ -
clang -cc1 version 14.0.6 based upon LLVM 14.0.6 default target x86_64-redhat-linux-gnu
ignoring nonexistent directory "/data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc10/external/gcc/10.3.0-84898dea653199466402e67d73657f10/lib/gcc/x86_64-redhat-linux-gnu/10.3.0/../../../../x86_64-redhat-linux-gnu/include"
ignoring nonexistent directory "/include"
#include "..." search starts here:
#include <...> search starts here:
 /data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc10/external/gcc/10.3.0-84898dea653199466402e67d73657f10/lib/gcc/x86_64-redhat-linux-gnu/10.3.0/../../../../include/c++/10.3.0
 /data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc10/external/gcc/10.3.0-84898dea653199466402e67d73657f10/lib/gcc/x86_64-redhat-linux-gnu/10.3.0/../../../../include/c++/10.3.0/x86_64-redhat-linux-gnu
 /data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc10/external/gcc/10.3.0-84898dea653199466402e67d73657f10/lib/gcc/x86_64-redhat-linux-gnu/10.3.0/../../../../include/c++/10.3.0/backward
 /data/cmsbld/jenkins/workspace/ib-run-pr-tests/testBuildDir/el8_amd64_gcc10/external/llvm/14.0.6-7e877bac6aa04ac511ab1c515ec417ab/lib64/clang/14.0.6/include
 /usr/local/include
 /usr/include
End of search list.
```

Reply:
I still need to see if it works properly when it is installed on cvmfs